test/browser/sauce-browsers.json

```diff
@@ -1,5 +1,12 @@
 [
   {
+    "browserName": "microsoftedge",
+    "platform": "WIN10",
+    "version": "20.10240"
```

Comment:
can we leave off `version`? normally sauce will default to `latest` when this is done, no?"